src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -249,6 +285,35 @@ class PackageDetailTab extends mixin(StoreMixin) {
     }
   }
 
+  handlePackageVersionChange(packageOpt) {
+    const packageVersion = packageOpt.id;
+    const originalPath = global.location.hash.split("?")[0];
+    const selectedVersion = `${originalPath}?version=${packageVersion}`;
+
+    global.location.replace(selectedVersion);
```

Comment:
Could we use react router tools to do this?
```
// To set a parameter:
<Link to={{
    pathname: router.location.pathname,
    query: {
        ...(router.location.query),
        myProp: 'foo',
    }
 }} />
```

Reply:
`<Link />` returns a component which expects a click, so in that case doesn't work for our case. https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/modules/Link.js

Reply:
Ok, maybe we can do the same as this https://github.com/dcos/dcos-ui/blob/c693793411634bbd23e452648cb5f33dfdb20852/src/js/pages/catalog/PackagesTab.js#L95